INSTALL.md

```diff
@@ -107,30 +107,57 @@ then `rm -rf vendor/` to remove `client-go`'s copy of its dependencies.
 
 ### Glide
 
-[glide](https://github.com/Masterminds/glide) is another popular dependency
-management tool for go. Glide will manage your /vendor directory, but unlike
-godep, will not modify your $GOPATH (there's no equivalent of `godep restore` or
-`godep save`). To get the `client-go` dependency list into glide's format, you
-can run this series of commands:
+[Glide](https://github.com/Masterminds/glide) is another popular dependency
+management tool for Go. Glide will manage your /vendor directory, but unlike
+godep, will not use or modify your $GOPATH (there's no equivalent of
+`godep restore` or `godep save`).
+
+Generally, it's best to avoid Glide's many subcommands, favoring modifying
+Glide's manifest file (`glide.yaml`) directly, then running
+`glide update --strip-vendor`. First create a `glide.yaml` file at the root of
+your project:
+
+```yaml
+package: ( your project's import path ) # e.g. github.com/foo/bar
+import:
+- package: k8s.io/client-go
+  version: v2.0.0
+```
+
+Second, add a Go file that imports `client-go` somewhere in your project (Glide
```

Comment:
I tried it locally. Glide still added client-go to vendor/, but it didn't parse client-go/godeps.json, so all client-go's dependencies were missed in vendor/. So how about this:

Second, add a Go file that imports `client-go` somewhere in your project, otherwise client-go's dependencies will not be added to your procject's vendor/.